msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,626 @@
+
+1. Copy the **infra** folder into the new subfolder, and then delete the `azure.parameters.json` file from the new **tab** > **infra** folder.
+1. Move the **node_modules** and **src** folders into the new subfolder.
+1. Move the `.webappignore`, `package-lock.json`, `package.json`, `tsconfig.json`, and `web.config` files into the new subfolder. 
+
+    ```
+    |-- .vscode/
+    |-- appPackage/
+    |-- build
+    |-- env/
+    |-- infra/
+    |-- tab/
+    |-- |-- infra/
+    |-- |-- node_modules/
+    |-- |-- src/
+    |-- |-- .webappignore
+    |-- |-- package-lock.json
+    |-- |-- package.json
+    |-- |-- tsconfig.json
+    |-- |-- web.config
+    |-- .gitignore
+    |-- .localConfigs
+    |-- teamsapp.local.yml
+    |-- teamsapp.yml
+    ```
+
+1. In the `package.json` that you just moved to the tab folder, delete the script named `dev:teamsfx` from the `scripts` object. This script is added to a new `package.json` in the next step.
+1. Create a new file named `package.json` in the root of the project and add the following content:
+
+    ```
+    {
+        "name": "CombinedTabAndAddin",
+        "version": "0.0.1",
+        "author": "Contoso",
+        "scripts": {
+            "dev:teamsfx": "env-cmd --silent -f .localConfigs npm run start:tab",
+            "build:tab": "cd tab && npm run build",
+            "install:tab": "cd tab && npm install",
+            "start:tab": "cd tab && npm run start",
+            "test": "echo \"Error: no test specified\" && exit 1"
+        },
+        "devDependencies": {
+            "@microsoft/teamsfx-cli": "2.0.2-alpha.4f379e6ab.0",
+            "@microsoft/teamsfx-run-utils": "alpha",
+            "env-cmd": "^10.1.0",
+            "office-addin-dev-settings": "^2.0.3",
+            "ncp": "^2.0.0"
+        }
+    }
+    ```
+
+1. Change the `name`, `version`, and `author` properties, as needed.
+1. Open the `teamsapp.yml` file in the root of the project, find the line `ignoreFile: .webappignore`, and change it to `ignoreFile: ./tab/.webappignore`.
+1. Open the `teamsapp.local.yml` file in the root of the project, find the line `args: install --no-audit`, and change this to `args: run install:tab --no-audit`.
+1. Open **TERMINAL** in Visual Studio Code. Navigate to the root of the project and run `npm install`. A new `node_modules` folder and a new `package.lock.json` file are created in the project root. 
+1. Next run `npm run install:tab`. A new `node_modules` folder and a new `package.lock.json` file are created in the tab folder, if they aren't there already. 
+1. Verify that you can sideload the tab with the following steps:
+
+    1. Open Teams Toolkit. 
+    1. In the **ACCOUNTS** section, verify that you're signed in to Microsoft 365 account.
+    1. Select **View** > **Run** in Visual Studio Code.
+    1. In the **RUN AND DEBUG** dropdown menu, select **Debug in Teams (Edge)**, and press F5. 
```

Comment:
```suggestion
    1. In the **RUN AND DEBUG** dropdown menu, select **Debug in Teams (Edge)** and press F5. 
```

Reply:
This change would violate the guidance in the style guide: https://styleguides.azurewebsites.net/StyleGuide/Read?id=2700

Reply:
You're correct. I'll accept the change.